### PR TITLE
EBPF error handling

### PR DESCRIPTION
**Pull request by dvdlevanon**

### CODE REVIEW

File: tap/tlstapper/bpf/openssl_uprobes.c [hunk 1]
```diff
@@ -198,9 +194,8 @@ static __always_inline void ssl_uretprobe(struct pt_regs *ctx, struct bpf_map_de
 	
 	struct ssl_info *infoPtr = bpf_map_lookup_elem(map_fd, &id);
 	
-	if (infoPtr == NULL) {
-		char msg[] = "Error getting ssl context info (id: %ld)";
-		bpf_trace_printk(msg, sizeof(msg), id);
+	if (infoPtr == 0) {
```
Comment (gadotroee) [inline tap/tlstapper/bpf/openssl_uprobes.c:197]: maybe mistake when merging the develop into this branch?

Comment (gadotroee) [inline tap/tlstapper/bpf/openssl_uprobes.c:197]: maybe mistake when merging the develop into this branch?

**Review by gadotroee — COMMENTED** [review]
added small comments

File: tap/passive_tapper.go [hunk 2]
```diff
@@ -245,9 +245,10 @@ func startPassiveTapper(opts *TapOpts, outputItems chan *api.OutputChannelItem)
 
 func startTlsTapper(extension *api.Extension, outputItems chan *api.OutputChannelItem, options *api.TrafficFilteringOptions) {
 	tls := tlstapper.TlsTapper{}
-	tlsPerfBufferSize := os.Getpagesize() * 100
+	chunksBufferSize := os.Getpagesize() * 100
```
Comment (nimrod-up9) [inline tap/passive_tapper.go:248]: Just a question: Why does the buffer size depends on the page size?

Comment (dvdlevanon) [reply tap/passive_tapper.go:262]: Perf and Ring buffers are define in number pages.
It is rounded up to the nearest multiple of the current page size.

There is more info about it here:
https://nakryiko.com/posts/bpf-ringbuf/

**Review by gadotroee — APPROVED** [review]

**Pull request merged**